src/editor/FluxQueryField.tsx

```diff
@@ -205,16 +199,21 @@ export default class FluxQueryField extends QueryField {
     }
   }, 500);
 
-  applyTypeahead = (editor: CoreEditor, suggestion: { text: any; type: string; deleteBackwards: any }): CoreEditor => {
+  applyTypeahead = (editor: CoreEditor, suggestion: any): CoreEditor => {
     const { typeaheadPrefix, typeaheadContext, typeaheadText } = this.state;
-    let suggestionText = suggestion.text || suggestion;
+
+    if (typeof suggestion['display'] !== 'undefined') {
+      // We have a flux function
```

Comment:
What other suggestions are there?

Reply:
When you click the mouse, (handleClickMenu). You got a different object than when you used enter and tab. On enter and tab, you got a 'display' field that came from the functions that were described in flux.ts. This was to convert that to the standard suggestion type format.

Reply:
Hmm, can we harmonize this so that mouse and keyboard event handlers turn events into accepted suggestion items?

Reply:
https://github.com/grafana/influxdb-flux-datasource/issues/58